Sources/WordPressShared/Utility/String+RemovingMatches.swift

```diff
@@ -15,7 +15,7 @@ extension String {
         do {
             regex = try NSRegularExpression(pattern: pattern, options: options)
         } catch {
-            DDLogError(("Error parsing regex: \(error)"))
+            WPSharedLogError("Error parsing regex: \(error)")
```

Comment:
Next step: we should deprecate this method in favour of one that `throw`s – printing an error to the log then doing the wrong thing is IMHO not great 😅